tox.ini

```diff
@@ -42,11 +42,22 @@ commands =
 
 [testenv:integrationtests]
 basepython = python3.8
+passenv = LANG TERM LANGUAGE LC_ALL LD_LIBRARY_PATH SOLC_BINARY BRANCH INTEGRATION_TESTS_REPO_URL
 deps =
 	-rrequirements/tests.txt
+setenv =
+	INT_TESTS_REPO_SSH_URL = git@github.com:gridsingularity/gsy-backend-integration-tests.git
+whitelist_externals =
```

Comment:
`whitelist_externals` has been deprecated and it's now `allowlist_externals`.

Reply:
Sure. I also thought about it but decided to stick with that previously implemented.